src/huggingface_hub/hub_mixin.py

```diff
@@ -1,6 +1,8 @@
 import json
 import os
 from pathlib import Path
+from safetensors import safe_open
+from safetensors.torch import save_file
```

Comment:
This should not be in base imports as they should be optional for the users. What you should do is import them only in a `if is_safetensors_available()` statement below, as done for `torch` (see L14). Since `huggingface_hub` is a collection of many helpers used in various situations, we want to limit the number of required dependencies.

Reply:
Thanks for the feedback, done in 859e230e299a4fa6d87d72a5718d6f478bcc4920